library/src/main/scala/scala/scalajs/js/annotation/internal/JSOptional.scala

```diff
@@ -18,5 +18,7 @@ import scala.annotation.meta._
  *
  *  Do not use this annotation yourself.
  */
-@field @getter @setter
+@field
+@getter
+@setter
```

Comment:
This is an undesirable change. scalafmt 0.5.2 will have an option not to affect the line grouping of annotations, IIUC.